### ISSUE TITLE
Please support Nokogiri >= 1.10.3

### ISSUE THREAD

**Issue by JunichiIto** [microsoftgraph/msgraph-sdk-ruby#44]
I want to update Nokogiri to 1.10.3 due to [CVE-2019-11068](https://github.com/sparklemotion/nokogiri/issues/1892), but this gem is preventing from it.

I understand [you are a small team](https://github.com/microsoftgraph/msgraph-sdk-ruby/issues/42), but I would like you to fix it ASAP for the vulnerability.

**Comment by JunichiIto** [comment]
My pull request might solve this issue:

https://github.com/microsoftgraph/msgraph-sdk-ruby/pull/45

**Comment by Younnah** [comment]
Hi @JunichiIto 

When I add `gem 'microsoft_graph'` in the Gemfile and I run `bundle install`, I have this error:

<img width="537" alt="Screenshot 2020-02-10 at 18 01 03" src="https://user-images.githubusercontent.com/26743683/74227871-a3df3400-4cbf-11ea-9201-ac0dcea7b674.png">

I was told that your pull request was about this problem...

Here is my stack:
ruby : ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin19]
ruby on rails : 5.2.3

I would like to use the gem as in this doc : https://docs.microsoft.com/fr-fr/outlook/rest/ruby-tutorial

Thank you for your help!

**Comment by JunichiIto** [comment]
@Younnah Your microsoft_graph version seems 0.1.1 but the latest version is 0.1.3 ([ref.](https://rubygems.org/gems/microsoft_graph)), and it might solve the problem. Could you try this?

```ruby
# Gemfile
gem 'microsoft_graph', '~> 0.1.3'
```

```
bundle update microsoft_graph
```

**Comment by Younnah** [comment]
@JunichiIto thank you for your answer!
I tried, but I use the 1.10.4 nokogiri gem version and I have the same issue.
```Bundler could not find compatible versions for gem "nokogiri":
  In Gemfile:
    nokogiri (= 1.10.4)

    microsoft_graph (~> 0.1.3) was resolved to 0.1.3, which depends on
      nokogiri (~> 1.8.0)
```

Btw, add `gem 'microsoft_graph', github: 'microsoftgraph/msgraph-sdk-ruby'` solved my problem!

**Comment by JunichiIto** [comment]
@Younnah Ah, I noticed my PR has not been released yet. (0.1.3 was released in 2017!)

> Btw, add gem 'microsoft_graph', github: 'microsoftgraph/msgraph-sdk-ruby' solved my problem!

Yep, your solution seems okay 👌
